ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/productteaser/v1/productteaser/actions.html

```diff
@@ -15,11 +15,11 @@
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
 <template data-sly-template.actions ="${@ product}">
     <div class="productteaser__cta" data-sly-test="${product.callToAction != null}">
-        <button data-sly-test="${product.callToAction == 'add-to-cart'}" data-action="addToCart" data-item-sku="${product.sku}"
-                class="button__root_highPriority button__root clickable__root button__filled" type="button">
+        <button data-sly-test="${product.callToAction == product.CALL_TO_ACTION_TYPE_ADD_TO_CART}" data-action="${product.callToActionCommand}" data-item-sku="${product.sku}"
```

Comment:
This will not resolve the defect for customers that have overwritten the `actions.html`. We should try to find a solution that works for all customers.